markdown/ch7.markdown

```diff
@@ -401,24 +430,41 @@ for ax in (ax0, ax1, ax2):
 
 ```
 
-Oops! Even though we started closer to the original image, we failed to
+Even though we started closer to the original image, we failed to
 recover the correct rotation. This is because optimization techniques can get
 stuck in local minima, little bumps on the road to success, as we saw above
 with the shift-only alignment. They can therefore be quite sensitive to the
 starting parameters.
 
+## Avoiding local minima with basin hopping
+
+A 1997 algorithm devised by David Wales and Jonathan Doyle [^basinhop], called
```

Comment:
I wish I knew this during my PhD; I had recurring issues with registration that messed up many experiments to the point of exasperation.